tests/human_perception/test_top_down_pose_estimation.py

```diff
@@ -33,7 +33,7 @@ def fixture():
         image_array=image_array, output_path=video_path, disable_log=True)
 
 
-def test_mmdet_detector():
+def test_pose_detector():
```

Comment:
What about splitting them into two functions, `test_mmpose_topdown_estimator` and `test_mediapipe_estimator`?
If one fails, the other goes on.